src/navbar/menu.styles.tsx

```diff
@@ -0,0 +1,207 @@
+
+    ${MediaQuery.MaxWidth.tablet} {
+        display: none;
+    }
+`;
+
+export const MobileWrapper = styled.ul`
+    display: none;
+    list-style: none;
+
+    ${MediaQuery.MaxWidth.tablet} {
+        border-left: 4px solid ${Color.Primary};
+        display: flex;
+        flex-direction: column;
+        padding-bottom: 1rem;
+    }
+`;
+
+// =============================================================================
+// LINK ITEMS
+// =============================================================================
+export const LinkItem = styled.li`
+    position: relative;
+    display: flex;
+    align-items: center;
+    :not(:last-of-type) {
+        margin-right: 1rem;
+    }
+
+    ${MediaQuery.MaxWidth.tablet} {
+        width: 100%;
+
+        :not(:last-of-type) {
+            margin-right: 0;
+        }
+    }
+`;
+
+export const Link = styled(Text.Hyperlink.Small)<StyleProps>`
+    ${TextStyleHelper.getTextStyle("H6", 400)}
+    display: flex;
+    position: relative;
+    align-items: left;
+    text-align: left;
+    color: ${Color.Neutral[1]};
+
+    /* font-style: normal;
+    //font-weight: 400; */
+    font-size: 14px;
+    line-height: 20px;
+
+    margin-right: 16px;
+    padding-right: 16px;
+
+    overflow: hidden;
+    display: -webkit-box;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
+
+    /* identical to box height, or 186% */
+    letter-spacing: 0.12px;
+
+    :active,
+    :focus {
+        color: ${Color.Primary} !important;
+    }
+    :hover {
+        color: ${Color.Accent.Light[1]};
+    }
+
+    ${MediaQuery.MaxWidth.tablet} {
+        ${(props) => {
+            return TextStyleHelper.getTextStyle(
+                "H6",
+                props.$selected ? "bold" : "regular"
+            );
+        }}
+        ${TextStyleHelper.getTextStyle("H6", "bold")}
+        /* font-weight: bold !important; */
+        width: 100%;
+        padding: 0.625rem 1rem;
+        text-align: left;
+        align-items: flex-start;
+        font-size: 16px !important;
+        line-height: 18px !important;
+        overflow: hidden;
+        display: -webkit-box;
+        -webkit-line-clamp: 2;
+        -webkit-box-orient: vertical;
+    }
+`;
+
+export const LinkIndicator = styled.div`
+    position: absolute;
+    bottom: 0;
+    height: 4px;
+    left: 0.5rem;
+    right: 0.5rem;
+    background-color: ${Color.Primary};
+
+    ${MediaQuery.MaxWidth.tablet} {
+        left: 0;
+        right: unset;
+        top: 0;
+        bottom: 0;
+        height: 100%;
+        width: 4px;
+        background-color: ${Color.Accent.Light[1]};
+    }
+`;
+
+export const MenuBar = styled.ul`
+    width: 100%;
+    overflow: auto;
+    display: grid;
+    margin-top: 0rem;
+    left: 0;
+    top: 102%;
+    min-width: 250px;
+    /* N8 #FFFFFF */
+    position: absolute;
+    overflow-x: hidden;
+
+    /* N8 #FFFFFF */
+    background: #ffffff;
+    box-shadow: 0px 2px 8px rgba(75, 83, 159, 0.24);
```

Comment:
use built-in theme colors instead of hardcoding the hexes or rgba values

usually the Figma comments will tell you which color to use. for example N8 means `Color.Neutral[8]`